Makefile

```diff
@@ -42,19 +42,30 @@ test_processors: test_processors.S hexagon_vm.h Makefile
 	${CC} ${CFLAGS} -o $@ $< ${GUEST_LDFLAGS}
 
 .PHONY: test FORCE
+
+TEST_CASES=\
+	first \
+	test_mmu \
+	test_interrupts \
+	test_processors \
+	$()
+
 test: minivm first test_mmu test_interrupts test_processors FORCE
 	qemu-system-hexagon -M SA8775P_CDSP0 ${QEMU_OPTS} -device loader,addr=${GUEST_ENTRY},file=./first -kernel ./minivm
 	qemu-system-hexagon -M SA8775P_CDSP0 ${QEMU_OPTS} -device loader,addr=${GUEST_ENTRY},file=./test_mmu -kernel ./minivm
 	qemu-system-hexagon -M SA8775P_CDSP0 ${QEMU_OPTS} -device loader,addr=${GUEST_ENTRY},file=./test_interrupts -kernel ./minivm
 	qemu-system-hexagon -M SA8775P_CDSP0 ${QEMU_OPTS} -device loader,addr=${GUEST_ENTRY},file=./test_processors -kernel ./minivm
 
-.PHONY: dbg
+.PHONY: dbg install
 dbg: FORCE
 	lldb -o 'file ./minivm' -o 'target modules add ./vmlinux' -o 'target modules load -s 0 --file ./vmlinux' -o 'gdb-remote localhost:1234' ${LLDB_OPTS}
 
 minivm.bin: minivm
 	${OBJCOPY} -O binary $< $@
 
+install: minivm minivm.bin $(TEST_CASES)
+	 install -D $^ $(DESTDIR)$(bindir)
```

Comment:
Should we default DESTDIR to something like "install" if it is not defined? Or even error out?

Also, bindir doesn't seem to be defined?

Reply:
If `DESTDIR` is substituted as the empty string then the implicit behavior would be that the user wants them installed to the root.  It's similar-but-not-quite-the-same as the `prefix` / `--prefix`.

Hmm, I guess, now that I'm thinking about it - `prefix` is the Right Thing to do here, so I'll switch to that.

See https://www.chiark.greenend.org.uk/doc/make-doc/make.html/Makefile-Conventions.html#Directory-Variables and https://www.chiark.greenend.org.uk/doc/make-doc/make.html/Makefile-Conventions.html#DESTDIR

Reply:
> Also, bindir doesn't seem to be defined?

I'll fix this